README.md

```diff
@@ -0,0 +1,11 @@
+# WordPress VIP Node.js Skeleton
+
+Welcome to WordPress VIP! This repository is a starting point for building a Node.js application on WordPress VIP. It implements the basic requirements needed to successfully deploy and run an application on the platform. The entrypoint [`/src/index.js`][./src/index.js] is heavily annotated with comments that explain these requirements.
```

Comment:
This line's not getting formatted correctly, and I think it's because you used double square brackets for both of them, instead of []( )